app/views/spree/admin/product_questions/pending.html.erb

```diff
@@ -15,7 +15,7 @@
       </tr>
     </thead>
     <tbody>
-      <% @collection.each do |product| %>
+      <% @collection.uniq.each do |product| %>
```

Comment:
why do we need uniq here?

Reply:
For multiple questions for same product, the product was displayed multiple times.